packages/core/src/bin/commands/dev.ts

```diff
@@ -292,6 +292,8 @@ export async function dev({ cliOptions }: { cliOptions: CliOptions }) {
   const namespace =
     cliOptions.schema ?? process.env.DATABASE_SCHEMA ?? "public";
 
+  globalThis.PONDER_NAMESPACE_BUILD = namespace;
```

Comment:
Is there a reason we don't use namespaceCompile?

Reply:
There are different rules for start vs dev, and namespaceCompile implements the start rules